services/tinlake/actions.ts

```diff
@@ -473,13 +522,13 @@ export async function redeem(
     return loggedError(e, `Could not redeem ${trancheType}.`, '')
   }
 
-  return { status: 0 }
+  return { status: 0 } as any
```

Comment:
It's a bit of a hack, I added a `.receipt()` method to `PendingTransaction` in tinlake.js, in response to one of your comments there, but then I'm missing that key here now..

Reply:
How can we type this better?

Reply:
Okay, I can't easily fix this, because the published tinlake v2 also contains the `.receipt()` already... So I propose we leave it as is but add a TODO to fix this. We need to discuss the .receipt() syntax still anyhow. That's just been delayed because we haven't merged the Tinlake.js rewrite that ,because we weren't able to test it.

Reply:
Sounds good!